pkg/s3presign/presign_test.go

```
@@ -423,3 +424,266 @@ func TestGeneratePresignedGetURL(t *testing.T) {
+			name:    "invalid INI data returns error",
+			data:    "this is not valid ini or arn data",
+			wantErr: true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.envTokenFile != "" {
+				t.Setenv("AWS_WEB_IDENTITY_TOKEN_FILE", tt.envTokenFile)
+			} else {
+				os.Unsetenv("AWS_WEB_IDENTITY_TOKEN_FILE")
```

golangci-lint (2.11.4)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)